cfme/utils/template/base.py

```diff
@@ -234,19 +241,46 @@ def deploy_template(self):
 
     @log_wrap("download image locally")
     def download_image(self):
+        suffix = re.compile(
+            r'^.*?[.](?P<ext>tar\.gz|tar\.bz2|\w+)$').match(self.image_name).group('ext')
         # Check if file exists already:
         if path.isfile(self.local_file_path):
             logger.info('Local image found, skipping download: %s', self.local_file_path)
-            return True
+            if suffix not in ['zip', 'tar.gz']:
```

Comment:
`suffix` uses `self.image_name`, whereas this is operating off of `self.local_file_path` - I think that you need to compile the suffix regex from `self.local_file_path` to be specific here that you're checking the _downloaded_ file that was found has been unpacked

Reply:
Actually local_file_path uses image_name anyway. I don't need whole path I think?
local_file_path = project_path.join(self.image_name).strpath
I will rather should everywhere use image_name than local_file_path.